File: javaScript/level_0/odd-even.js
/*
 * Guess the odd or even number
 */
const prompt = require("prompt-sync")();
const input = prompt("Input a number : ");

function guessOddOrEvenNumber() {
  if (input % 2 == 0) {
    console.log("EVEN");
  } else {
    console.log("ODD");
  }
}

guessOddOrEvenNumber();

File: python/level_0/odd_even.py
"""
    Guess the odd or even number
"""

input_number = int(input("Input a number : "))


def guess_odd_or_even_number():
    if input_number % 2 == 0:
        print("EVEN")
    else:
        print("ODD")


guess_odd_or_even_number()
